.revdeprefs.yaml

```diff
@@ -0,0 +1,6 @@
+- insightsengineering/tern
+- insightsengineering/tern.gee
+- insightsengineering/tern.mmrm
+- insightsengineering/tern.rbmi
+- insightsengineering/scda.test@remotes_tern
```

Comment:
```suggestion
- insightsengineering/scda.test
```
update this line, if https://github.com/insightsengineering/scda.test/pull/171 is merged

Reply:
I think it would be better to remove Remotes field from `tern.rbmi`. WDYT?

Reply:
hi @pawelru , if we can merge this in https://github.com/insightsengineering/scda.test/pull/171, i think we can change it to  insightsengineering/scda.test